### PR TITLE
Added implementation in common lisp

### PR DESCRIPTION
**Pull request by NicoSimoski**
- with system-definition (asdf)
- with unit-tests (parachute)

### CODE REVIEW

File: common-lisp/parrot-refactoring-kata.asd [hunk 1]
```diff
@@ -0,0 +1,22 @@
+;;;; parrot-refactoring-kata.asd
+
+(asdf:defsystem "parrot-refactoring-kata"
+  :description ""
+  :author "King Arthur king@roundtable.org"
```
Comment (emilybache) [inline common-lisp/parrot-refactoring-kata.asd:5]: For this one you probably want to credit Monty Python (look up the dead parrot sketch)

**Comment by emilybache** [comment]
Thankyou!

File: common-lisp/parrot-refactoring-kata.asd (hunk 1 above)
Comment (NicoSimoski) [reply common-lisp/parrot-refactoring-kata.asd:5]: I am not very familiar with copyrights but wouldn't it be nicer to credit them in the readme of this kata?

Comment (emilybache) [reply common-lisp/parrot-refactoring-kata.asd:5]: It's just a bit of fun! Most languages don't have a space in the build file to credit an author, so none of the others have this. It's really not important.

Comment (NicoSimoski) [reply common-lisp/parrot-refactoring-kata.asd:5]: Sorry, I just didn't get your intention. :-D
How do you like "Monty Python monty.python@legendary.fun"?

Comment (emilybache) [reply common-lisp/parrot-refactoring-kata.asd:5]: Yes, sounds good!

**Pull request merged**